src/renderer/components/ParentCryptoCurrencyIcon.js

```diff
@@ -29,7 +29,7 @@ const ParentCryptoCurrencyIconWrapper: ThemedComponent<{
   line-height: ${p => (p.bigger ? "18px" : "18px")};
   font-size: ${p => (p.bigger ? "12px" : "12px")};
   > :nth-child(2) {
-    margin-top: ${p => (p.bigger ? "-10px" : "-8px")};
+    margin-top: ${p => (p.bigger ? "-15px" : "-14px")};
```

Comment:
hi :wave: i changed this file in PR #175 we found that the clip-path on the currency svg icon was deoptimising browser scroll forcing it to repaint and hindering ui performance.
:boom:

Reply:
oh my, well I'll remove the polish of that part since it's going to be 💥 soon